include/fdreadoutlibs/wibeth/WIBEthFrameProcessor.hpp

```diff
@@ -1,46 +1,30 @@
 /**
- * @file WIBEthProcessor.hpp WIBEth specific Task based raw processor
- * @author Giovanna Lehmann Miotto (giovanna.lehmann@cern.ch)
+ * @file WIBEthFrameProcessor.hpp WIBEth specific Task based raw processor
+ * @author Adam Abed Abud (adam.abed.abud@cern.ch)
```

Comment:
no authors in lib/spec headers please. Git blame is there for these. :)
The TPG being a major standalone algorithm implementation with Phil's name is fine.